camp_fin/models.py

```diff
@@ -5,24 +5,26 @@
 from dateutil.rrule import rrule, MONTHLY
 from django.conf import settings
 from django.utils.translation import ugettext as _
+from django.utils import timezone
```

Comment:
Most of these changes are formatting. The material ones are:

- Increasing the max length of name fields.
- Adding an automatic value to `date_added`.